tracing-flame/src/lib.rs

```diff
@@ -301,11 +322,20 @@ where
         let samples = self.time_since_last_event();
 
         let first = ctx.span(id).expect("expected: span id exists in registry");
+
+        if self.config.filter_empty && first.from_root().count() == 0 {
+            return;
+        }
```

Comment:
It _might_ also be worth having the option to emit a dummy name like "[no span]" or something in this case, probably as another config option? But we don't have to do that right now.